### ISSUE TITLE
Not able to start rancher-nfs in a rancher-server setup using self signed certs.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#7270]
Rancher Version:v1.3.0-rc2 using self signed certs.

rancher-server is started with the following command - sudo docker run -d --restart=unless-stopped --name=rancher-server -v :/var/lib/rancher/etc/ssl/ca.crt rancher/server:v1.3.0-rc2

It is hosted behind nginx server that acts as a reverse proxy.

On the host , certs have to be copied to /var/lib/rancher/etc/ssl/ca.crt before running the host registration url which will already include -v /var/lib/rancher:/var/lib/rancher

Create a cattle environment with few hosts:

Deploy rancher-nfs.

nfs-driver containers fails to start with the following errors:
```
1/3/2017 2:13:18 PMtime="2017-01-03T22:13:18Z" level=info msg=Running
1/3/2017 2:13:18 PMtime="2017-01-03T22:13:18Z" level=info msg=Starting
1/3/2017 2:13:18 PMtime="2017-01-03T22:13:18Z" level=fatal msg="Get https://sangee-test.rancher.space/v2-beta: x509: certificate signed by unknown authority"
1/3/2017 2:15:34 PMtime="2017-01-03T22:15:34Z" level=info msg=Running
1/3/2017 2:15:34 PMtime="2017-01-03T22:15:34Z" level=info msg=Starting
1/3/2017 2:15:34 PMtime="2017-01-03T22:15:34Z" level=fatal msg="Get https://sangee-test.rancher.space/v2-beta: x509: certificate signed by unknown authority"
```

**Comment by LLParse** [comment]
This applies to all storage plugins (NFS/EFS/EBS) and Swarm as well.

**Comment by LLParse** [comment]
https://github.com/rancher/rancher-catalog/pull/403

Available in rancher-catalog `v1.4.0-rc2` branch

**Comment by galal-hussein** [comment]
Tested and verified that Storage plugins (rancher-efs, rancher-nfs, rancher-ebs) with Rancher v1.4.0-rc2 self signed certificate, and Swarm on Rancher master build are working.
